added release notes up to 10.17.180.0 (#188)
SoftwareAG/c8y-releasenotes: content/release-10-18-0/devices-10-18-0.md

<td>10.17.25.0</td>
</tr>

<tr>
<td>LWM2M</td>
<td>A LWM2M device can now completely be deleted from the devices list without an additional clean-up operation. This is done in the same way as any other device is deleted, that is, via the delete icon in the respective row.</td>
<td>DM-867</td>
<td>10.17.173.0</td>
</tr>

<tr>
<td>LWM2M</td>
<td>If event logging is enabled for the LWM2M microservice from the configuration file of the LWM2M agent, the agent now creates new logs as events with detailed information about the reason of failure when a device operation is not executed due to the device connectivity.</td>
<td>DM-231</td>
<td>10.17.37.0</td>
</tr>

<tr>
<td>OPC UA</td>

> <td>The OPC UA device gateway can now be run with Thin Edge. The gateway device is automatically created as child of the Thin Edge device. For more information on the configurations, check the Thin Edge section in <a href="https://cumulocity.com/guides/10.18.0/protocol-integration/opcua/#gateway-register" class="no-ajaxy">OPC UA > Gateway configuration and registration <a/>in the <i>Protocol integration guide</i>.</td>

Author: That is probably since it is not yet there :-). That is why I would leave this vague phrasing.

> <td>The OPC UA device gateway can now be run with Thin Edge. The gateway device is automatically created as child of the Thin Edge device. For more information on the configurations, check the Thin Edge section in <a href="https://cumulocity.com/guides/10.18.0/protocol-integration/opcua/#gateway-register" class="no-ajaxy">OPC UA > Gateway configuration and registration <a/>in the <i>Protocol integration guide</i>.</td>

Author: ```suggestion
<td>The OPC UA device gateway can now be run with Thin Edge. The gateway device is automatically created as a child of the Thin Edge device. For more information on the configurations, refer to the Thin Edge section in <a href="https://cumulocity.com/guides/10.18.0/protocol-integration/opcua/#gateway-register" class="no-ajaxy">OPC UA > Gateway configuration and registration<a/> in the <i>Protocol integration guide</i>.</td>
```

<td>10.17.16.0</td>
</tr>

<tr>
<td>Loriot</td>

> <td>Events created by the Loriot agent when sending uplink requests are now considered as device requests and provide connectivity status and availability monitoring.</td>

Reviewer: ```suggestion
<td>Events created by the Loriot agent when sending uplink requests are now considered device requests and provide connectivity status and availability monitoring.</td>
```

Author: I really stumble over this construction, but I trust you that it is correct. Sounds very weird to me which is why I left the "as" in:-).

<td>DM-2139</td>
<td>10.17.170.0</td>
</tr>

<tr>
<td>LWM2M</td>

> <td>During the LWM2M client registration, if the agent failed to find the registered device from the database due to a communication problem, an internal object related to the device managed object was automatically removed by the agent. Afterwards, LWM2M clients always failed to connect. This is now fixed and the internal object is only removed if the actual device managed object does not exist in the database.</td>

Reviewer: ```suggestion
<td>During the LWM2M client registration, if the agent failed to find the registered device in the database due to a communication problem, an internal object related to the device managed object was automatically removed by the agent. Afterwards, LWM2M clients always failed to connect. This is now fixed and the internal object is only removed if the actual device managed object does not exist in the database.</td>
```

we had this entry before, not sure about the preposition change.

Author: I also considered this but I´m not sure if it is "the device from the database" or "find the device in the database" which is why I left it as is.

<td>DM-2131</td>
<td>10.17.178.0</td>
</tr>

<tr>
<td>LWM2M</td>

> <td>The LWM2M device type can freely be chosen by the user. Therefore, devices which do not have the type "c8y_lwm2m" but still are LWM2M devices now also show the LWM2M bootstrap parameters view.</td>

Reviewer: what about passive -> active in the first sentence?
(The user can freely choose the LWM2M device type)

Author: Since the focus is on the device type here, I would leave this. What about 
"Since the LWM2M device type can freely be chosen by the user, devices..."

Reviewer: sure :)
